src/isIsogram.test.js

```diff
@@ -6,4 +6,24 @@ describe('isIsogram', () => {
   it(`should be declared`, () => {
     expect(isIsogram).toBeInstanceOf(Function);
   });
+
+  it(`should return boolean: 'false' when input value is empty`, () => {
```

Comment:
I think " return 'true' / 'false' "  is enough

Reply:
fixed